themebuilder/modules/less-template-loader.js

```diff
@@ -14,7 +14,7 @@ const createModifyVars = modifyVars => {
 };
 
 const addSwatchClass = (less, swatchSelector, modifyVars) => {
-    if(!swatchSelector) return less;
+    if(!swatchSelector) return less + createModifyVars(modifyVars);
```

Comment:
I suggest moving ```less + createModifyVars(modifyVars)``` to the local variable.
```suggestion
    const lessWithVars = less + createModifyVars(modifyVars);
    if(!swatchSelector) return lessWithVars;
    return swatchSelector + "{" + lessWithVars + "}";
```